addons/website/models/assets.py

```diff
@@ -46,8 +47,10 @@ def _get_custom_attachment(self, custom_url, op='='):
         See web_editor.Assets._get_custom_attachment
         Extend to only return the attachments related to the current website.
         """
+        if not self.env.user.has_group('website.group_website_designer'):
+            raise werkzeug.exceptions.Forbidden()
```

Comment:
I told you to use that group/forbidden form but your previous [implementation](https://github.com/odoo/odoo/compare/08cde3e22cd33b89c1cef8c4173b2d14efc5e8f5..e8dc234a6511e68fe5a0b8530859daae6833b454) was better for a bugfix in stable (group/sudo).

Indeed this will break potentiel customization that would do a `.sudo()._get_custom_attachment()` for users that would not be designer.